gitlab/config/201-clusterrole.yaml

```diff
@@ -62,6 +62,8 @@ rules:
   - get
   - list
   - watch
+  # Webhook controller needs it to update "sources-webhook-certs" contents
```

Comment:
nit: we should have webhook-specific secrets to avoid things clobbering each other.  This may need fixing elsewhere, but it's something I've started trying to do.